app/assets/javascripts/cas_client.js

```diff
@@ -0,0 +1,5 @@
+//= require jquery
+//= require rails-ujs
+//= require_self
+//= require_tree .
+//= require decidim/cas_client/hide_show_login
```

Comment:
This require is not needed since `require_tree` actually requires every javascript file inside the folder
```suggestion
```